src/displayapp/screens/WatchFaceTerminal.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <lvgl/src/lv_core/lv_obj.h>
+#include <chrono>
+#include <cstdint>
+#include <memory>
+#include "displayapp/screens/Screen.h"
+#include "components/datetime/DateTimeController.h"
+
+namespace Pinetime {
+  namespace Controllers {
+    class Settings;
+    class Battery;
+    class Ble;
+    class NotificationManager;
+    class HeartRateController;
+    class MotionController;
+  }
+
+  namespace Applications {
+    namespace Screens {
+
+      class WatchFaceTerminal : public Screen {
+      public:
+        WatchFaceTerminal(DisplayApp* app,
+                          Controllers::DateTime& dateTimeController,
+                          Controllers::Battery& batteryController,
+                          Controllers::Ble& bleController,
+                          Controllers::NotificationManager& notificatioManager,
+                          Controllers::Settings& settingsController,
+                          Controllers::HeartRateController& heartRateController,
+                          Controllers::MotionController& motionController);
+        ~WatchFaceTerminal() override;
+
+        void Refresh() override;
+
+        void OnObjectEvent(lv_obj_t* pObj, lv_event_t i);
+
+      private:
+        char displayedChar[8];
```

Comment:
Updates like done in WatchFaceDigital cleanup

```suggestion
        uint8_t displayedHour = -1;
        uint8_t displayedMinute = -1;
        uint8_t displayedSecond = -1;
```